src/main/kotlin/no/nav/familie/tilbake/api/DokumentRestService.kt

```diff
@@ -18,10 +19,18 @@ import java.util.UUID
 class DokumentRestService(val varselbrevService: VarselbrevService,
                           val manueltVarselbrevService: ManueltVarselbrevService) {
 
-    @GetMapping("/forhandsvis-varselbrev",
+    @GetMapping("/forhandsvis-manueltVarselbrev",
```

Comment:
Eg tror at me bør ha 1 endepunkt for forhåndsvisning av alle breva sendt frå "Send melding" i høgremenyen på behandling. D.v.s. Manuelt varselbrev, Innhent dokumentasjon og Korrigert varselbrev. Og eventuelt fritekstbrev om det blir aktuelt. For frontend er det mykje enklare om disse har same endepunkt.
Sjekk BrevRestTjeneste#forhåndsvisBrev for korleis det er gjort i fptilbake.

Reply:
Følger opp dette ifm. de andre brevene.